### PR TITLE
Update Hart footer

### LINKED ISSUE #1641: Update Hart council footer
Spoke to Amy at Hart council and they have updated their footer. We need to update the FMS Harts council footer too.

screen shot 2017-02-15 at 14 39 52

This is their old footer.
screen shot 2017-02-15 at 14 38 38

### PR DESCRIPTION
**Pull request by pezholio**
Fixes #1641 

Now looks like this:

download 1

### CODE REVIEW

**Comment by pezholio** [comment]
Beautiful, thanks. I hear you on the `target=_blank`s too. They were a hangover from a copy paste job from Hart's site.

**Comment by pezholio** [comment]
@dracos - These issues should all be sorted 👍

File: templates/web/hart/footer.html [hunk 1]
```diff
@@ -44,54 +44,64 @@
...
+    <footer id="hart-footer">
+      <div id="footer-inner" class="container">
+        <div id="footer-row-1" class="clearfix row">
+          <p class="desk-only">
+            <strong>©</strong>
+            <a href="http://www.hart.gov.uk/disclaimer" title="More information on the Disclaimer" class="hart">Hart District Council</a>
+            <a href="http://www.gov.uk/" title="GOV.UK">GOV.UK</a>
+          </p>
+          <ul id="footer-nav">
+              <li class="footer-link"> <a href="http://www.hart.gov.uk/Accessibility">Accessibility</a> </li>
```
Comment (dracos) [inline templates/web/hart/footer.html]: I'm not sure what .footer-link is for?

File: templates/web/hart/footer.html [hunk 2]
```diff
@@ -44,54 +44,64 @@
...
+              </a>
+            </li>
+            <li>
+              <a href="https://www.instagram.com/hartcouncil/">
+                <img alt="Instagram" src="/cobrands/hart/instagram-logo.png" style="width: 50px; height: 50px;">
+              </a>
+            </li>
+            <li class="web-chat">
+              <a href="http://www3.basingstoke.gov.uk/ccmwa/chat/a6f6143d-407f-4f99-b673-6feb98b54815">
+                <span class="desk-only web-chat-label">Web Chat</span>
```
Comment (dracos) [inline templates/web/hart/footer.html]: Or .web-chat-label?

File: web/cobrands/sass/_layout.scss [hunk 3]
```diff
@@ -27,7 +27,7 @@ $mappage-with-actions-sidebar-width: 58em;
     display: block;
 }
 .mob-only {
-    display: none;
+    display: none !important;
```
Comment (dracos) [inline web/cobrands/sass/_layout.scss]: A comment explaining why this is here (specificity in 3rd party CSS or similar) would be good.

File: web/cobrands/hart/hart.scss [hunk 4]
```diff
@@ -337,13 +337,18 @@ body.frontpage .table-cell .content {
 
     li {
       margin: 0 2px;
+
+      @media only screen and (max-width:48em) {
+        float: left;
+        margin-right: 5px;
```
Comment (dracos) [inline web/cobrands/hart/hart.scss]: What is this for? The icons appear practically identical without it?

File: web/cobrands/hart/hart.scss [hunk 5]
```diff
@@ -357,6 +345,16 @@ body.frontpage .table-cell .content {
       @media only screen and (max-width:48em) {
         float: none;
       }
+
+      a {
+        float: left;
```
Comment (dracos) [inline web/cobrands/hart/hart.scss]: Not sure what this is for either, it was okay before?

File: web/cobrands/hart/hart.scss [hunk 6]
```diff
@@ -234,130 +234,142 @@ body.frontpage .table-cell .content {
...
+    color: #fff;
+  }
+
+  p {
+    padding: 0;
+    margin: 0;
+  }
+
+  .row {
+    padding: 0 0.5em;
```
Comment (dracos) [inline web/cobrands/hart/hart.scss]: This is overridden on rows 1 and 3 by the specificity of the ID padding.

**Comment by dracos** [comment]
The two 'outdated' comments are not, thanks GitHub.

File: web/cobrands/hart/hart.scss (hunk 4 above)
Comment (pezholio) [inline web/cobrands/hart/hart.scss]: Took me a while to remember myself, but without floating the order appears incorrectly, and the spacing is uneven:

## Without

without

## With

with

**Comment by pezholio** [comment]
@dracos All sorted now I think 👍

**Comment by dracos** [comment]
The white gap is still there! It's just a default margin-top on footer that needs overriding, not sure if there's a reason you keep leaving it? :) Other than that, looks good.

**Comment by pezholio** [comment]
Finally(!) sorted this. Sorry for the delay 😄

**Comment by dracos** [comment]
Please remember to use `[Hart]` at the start of the commit message when squashed :)

**Pull request merged**